Documentation/network/lb-ipam.rst

```diff
@@ -429,3 +429,49 @@ for the network and broadcast addresses respectively.
     $ kubectl -n example get svc                
     NAME           TYPE           CLUSTER-IP     EXTERNAL-IP               PORT(S)          AGE
     service-blue   LoadBalancer   10.96.26.105   20.0.10.100,20.0.10.200   1234:30363/TCP   43s
+
+Sharing Keys
+------------
+
+Services can share the same ip or set of ips with other services. This is done by setting the ``io.cilium/lb-ipam-sharing-key`` annotation on the service.
```

Comment:
```suggestion
Services can share the same IP or set of IPs with other services. This is done by setting the ``io.cilium/lb-ipam-sharing-key`` annotation on the service.
```
Please change `ip` to `IP` here and in all other places. A bit of a nit, but it's required for consistency and for the build to pass.